app/src/main/java/ch/epfl/qedit/model/Quiz.java

```diff
@@ -65,6 +65,11 @@ public Builder(Quiz quiz) {
             this.treasureHunt = quiz.treasureHunt;
         }
 
+        public Builder setTreasureHunt(Boolean isTreasureHunt) {
```

Comment:
I don't think it is a really good idea to have that method here. That means that you can change the treasure hunt boolean in the middle of building a quiz, meaning that you could make the questions invalid.